submissions/Oleksandr-Heleta/popup/style.css

```diff
@@ -0,0 +1,213 @@
+* {
+    margin: 0;
+    padding: 0;
+
+}
+
+a {
+    text-decoration: none;
+    color: #757575;
+
+}
+
+li {
+    list-style: none;
+    cursor: pointer;
+
+}
+
+
+.popup_more-input,
+.popup_button-input {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+
+}
+
+
+img {
+    display: block;
+
+}
+
+.nav {
+    display: flex;
+    justify-content: flex-end;
+
+}
+
+.nav_item {
+    margin: 5px 10px;
+    padding: 5px;
+    max-height: 20px;
+
+}
+
+.nav_item:hover {
+    border: 1px solid #8bb6d6;
+    padding: 4px;
+
+}
+
+.nav_item:focus {
+    padding: 4px;
+    border: 2px solid black;
+
+}
+
+.popup_item:hover {
+    border: 1px solid #8bb6d6;
+    padding: 9px;
+
+}
+
+.popup_item:focus {
+    padding: 2px;
+    border: 2px solid black;
+}
+
+.nav-img {
+    max-height: 20px;
+
+}
+
+.popup {
+    display: none;
+    position: absolute;
+    top: 50px;
+    right: 0px;
+    z-index: 99;
+    overflow: auto;
+    max-width: 600px;
+    min-width: 300px;
+    height: 450px;
+    box-shadow: 0px 0px 20px 1px rgb(0 0 255 / 20%);
+    padding-top: 5px;
+
+}
+
+.popup_list {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+
+}
+
+.popup_item {
+    box-sizing: border-box;
+    padding: 10px;
+    max-width: 30%;
+
+}
+
+.popup_link {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+    text-align: center;
+
+}
+
+.popup_button {
+    position: relative;
+    min-width: 20px;
+
+}
+
+.popup_button-img {
+    height: 29px;
+    width: 29px;
+
+}
+
+.popup_button-label {
+    position: absolute;
+    top: 0;
+    left: 0;
+
+}
+
+.popup_button-input:focus~.popup_button-label {
+    border: 2px solid black;
+
+}
+
+.popup_img {
+    max-width: 100%;
+
+}
+
+#menu:checked~.popup {
+    display: flex;
+    flex-direction: column-reverse;
+
+}
+
+.popup_more_items {
+    display: none;
+
+}
+
+
+.popup_more_button {
+    position: relative;
+    display: block;
+    max-width: 500px;
+    padding: 5px;
+    margin: 10px;
+    height: 30px;
+    background-color: #d8eaf8;
+    color: #757575;
+    text-align: center;
+    cursor: pointer;
+
+}
+
+.popup_more_button:hover {
+    background-color: #aaa5a5;
+    color: #605959;
+
+}
+
+.popup_more-input:focus~.popup_more_button {
+    border: 2px solid black;
+
+}
+
+#more_less:checked~.popup_list>.popup_more_items {
+    display: block;
+
+}
+
+#more {
+    display: block;
+
+}
+
+#less {
+    display: none;
+
+}
+
+#more_less:checked~.popup_more_button #more {
+    display: none;
+
+}
+
+#more_less:checked~.popup_more_button #less {
+    display: block;
+
+}
+
+@media (max-width:400px) {
+    .popup {
+        position: absolute;
+        right: -50px;
+        max-width: 300px;
+        min-width: 250px;
+
+    }
+
+}
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)